crates/re_data_source/src/lib.rs

```diff
@@ -49,8 +45,11 @@ pub const SUPPORTED_MESH_EXTENSIONS: &[&str] = &["glb", "gltf", "obj"];
 
 pub const SUPPORTED_RERUN_EXTENSIONS: &[&str] = &["rrd"];
 
+pub const SUPPORTED_TEXT_EXTENSIONS: &[&str] = &["txt", "md"];
```

Comment:
We should support `.rs`, `.yaml`, `.json`, … etc too. Anything with the `text` mediatype basically.

Reply:
Hmmm... I'd rather not list every single language-specific file extension in existence, and we'd get very little value out of it: our rendering of plain text is pretty dull and non-interactive.

Same thing with structured formats (toml, json...), if we cannot make use of their structure, there's very little value in having opening them in plain-text, and I'd rather not list them all...

I think I'll open an issue to introduce a builtin catch-all `DataLoader` that accepts all things of this nature and logs them as markdown with the proper syntax hint: that way you at least get syntax coloring out of this.